server/routes/viewModels/taskLists/caTasks.js

```diff
@@ -119,7 +123,7 @@ module.exports = {
           href: '/hdc/victim/victimLiaison/',
           text: 'View/Edit',
         },
-        visible: curfewAddressApproved || bassChecksDone,
+        visible: !optedOut && validAddress,
```

Comment:
probably worth sticking this in a variable it's used in several places :/

Reply:
I chose to leave it that way because it was more expressive.

"If they're not opted out and they have a valid address"

Hard to think of a concise name for a variable to replace that.

Instead, although it means a small duplication (which is fine because it's easier to follow because the order is important and written out in full it's an exact reflection of the resulting view) it's probably clearer to pull out the small subset of tasks that ARE shown when opted out, and then the "opted in" case is cleaner.